external-providers/golang-external-provider/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/bombsimon/logrusr/v3"
+	"github.com/konveyor/analyzer-lsp/provider"
+	"github.com/konveyor/analyzer-lsp/provider/lib"
+	"github.com/konveyor/golang-external-provider/pkg/golang"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	configs, err := lib.GetConfig("/analyzer-lsp/provider_settings.json")
```

Comment:
This is hardcoded right now, to work in the container images. We need to re-do providers settings into two things:

1. Settings that the provider has that are hard coded
2. Settings to the analyzer binary that tells it where to look we have to figure out if we need some sort of expansion to the protocol to deal with a "session"

Reply:
I think users should only need to provide one config file to the engine, it should handle forwarding settings to the providers imo

Reply:
Captured in #142 I agree